site/_docs/powered_by.md

```diff
@@ -112,6 +112,12 @@ is a distributed JVM-Embeddable Database built on top of
 <a href="https://bookkeeper.apache.org/">Apache BookKeeper</a>.
 It uses Calcite as its SQL Planner.
 
+### MaxCompute
+
+<a href="https://www.alibabacloud.com">Alibaba</a>'s large-scale computing and storage system
```

Comment:
"Alibaba's big data computing and storage platform"

Reply:
updated.